sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj

```diff
@@ -20,10 +20,7 @@
     </Description>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" />
     <PackageReference Include="System.Text.Json" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Do we expect this to remain in this state? Or would will we be changing it back to project references in the future?

Reply:
I'd expect to switch back to project references.

Reply:
Why haven't we experience this issue between Blobs and Common?

Reply:
Because we have always shipped them together.